src/main/java/com/flippingutilities/controller/NewOfferEventPipelineHandler.java

```diff
@@ -30,6 +30,12 @@
      */
     @Subscribe
     public void onGrandExchangeOfferChanged(GrandExchangeOfferChanged offerChangedEvent) {
+        EnumSet<WorldType> currentWorldType = this.plugin.getClient().getWorldType();
+        EnumSet<WorldType> excludedWorldTypes = EnumSet.of(WorldType.LEAGUE, WorldType.DEADMAN);
+        if (!Collections.disjoint(currentWorldType, excludedWorldTypes)) {
+            return;
+        }
```

Comment:
Whitelist instead of blacklist?
We want pvp and normal worlds, not leagues, beta, dmm and any other custom type I assume